### PR TITLE
updated subpixel_classic's interface

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -53,9 +53,9 @@ jobs:
...
-          curl "https://asc-isisdata.s3.us-west-2.amazonaws.com/autocnet_test_data/D16_033458_1785_XN_01S046W.l1.cal.destriped.crop.cub" -o tests/test_subpixel_match/D16_033458_1785_XN_01S046W.l1.cal.destriped.crop.cub
-          curl "https://asc-isisdata.s3.us-west-2.amazonaws.com/autocnet_test_data/J04_046447_1777_XI_02S046W.l1.cal.destriped.crop.cub" -o tests/test_subpixel_match/J04_046447_1777_XI_02S046W.l1.cal.destriped.crop.cub
+          wget "https://asc-isisdata.s3.us-west-2.amazonaws.com/autocnet_test_data/B08_012650_1780_XN_02S046W.l1.cal.destriped.crop.cub" tests/test_subpixel_match/B08_012650_1780_XN_02S046W.l1.cal.destriped.crop.cub
+          wget "https://asc-isisdata.s3.us-west-2.amazonaws.com/autocnet_test_data/D16_033458_1785_XN_01S046W.l1.cal.destriped.crop.cub" tests/test_subpixel_match/D16_033458_1785_XN_01S046W.l1.cal.destriped.crop.cub
+          wget "https://asc-isisdata.s3.us-west-2.amazonaws.com/autocnet_test_data/J04_046447_1777_XI_02S046W.l1.cal.destriped.crop.cub" tests/test_subpixel_match/J04_046447_1777_XI_02S046W.l1.cal.destriped.crop.cub
```
Comment (jlaura) [inline .github/workflows/continuous_integration.yml:58]: 👍

File: autocnet/matcher/subpixel.py [hunk 2]
```diff
@@ -442,10 +442,8 @@ def subpixel_transformed_template(sx, sy, dx, dy,
...
 
-def subpixel_template_classic(sx, sy, dx, dy,
-                              s_img, d_img,
-                              image_size=(251, 251),
-                              template_size=(51,51),
+
+def subpixel_template_classic(reference_roi, moving_roi, affine=tf.AffineTransform(),
                               func=pattern_match,
                               **kwargs):
     """
```
Comment (jlaura) [inline autocnet/matcher/subpixel.py:449]: Need to get the doc string updated too once this is ready to merge.

File: autocnet/matcher/subpixel.py [hunk 3]
```diff
@@ -486,47 +484,32 @@ def subpixel_template_classic(sx, sy, dx, dy,
...
+    moving_clip = tf.warp(moving_clip, affine, order=3)
+    
+    print(moving_clip.var())
+    
+    if moving_clip.var() == 0:
         warnings.warn('Input ROI has no variance.')
         return [None] * 4
 
-    if (s_image is None) or (d_template is None):
+    if (ref_clip is None) or (moving_clip is None):
```
Comment (jlaura) [inline autocnet/matcher/subpixel.py:500]: @gsn9 `ref_clip` and `moving_clip` are also great variable names to choose from over image/template that was previously used (and is confusing).

File: autocnet/transformation/roi.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 from math import modf, floor
 import numpy as np
 
-from autocnet.transformation import dtypes
```
Comment (jlaura) [inline autocnet/transformation/roi.py:4]: Apologies that this snuck in.

**Review by jlaura — COMMENTED** [review]
Are the tests identical for this then? Not seeing any test changes is really nice, so I just want to confirm!

File: .github/workflows/continuous_integration.yml (hunk 1 above)
Comment (Kelvinrr) [reply .github/workflows/continuous_integration.yml:58]: ngl this was me throwing stuff at the wall to figure out how to get past the wget step, it seems like github runners can't hit our AWS storage?

**Comment by Kelvinrr** [comment]
@jlaura I'm getting weird unrelated test failures locally (unable to connect to db despite db running) and github tests are failing because of the wget/curl to AWS so I don't really know what to fix until I figure out those problems first.

File: README.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ We suggest using Anaconda Python to install Autocnet within a virtual environmen
 ## How to run the test suite locally
 
 1. Install Docker
-2. Get the Postgresql with Postgis container and run it `docker run --name testdb -e POSTGRES_PASSOWRD='NotTheDefault' -e POSTGRES_USER='postgres' -p 5432:5432 -d mdillon/postgis`
+2. Get the Postgresql with Postgis container and run it `docker run --name testdb -e POSTGRES_PASSOWRD='NotTheDefault' -e POSTGRES_USER='postgres' -p 35432:5432 -d mdillon/postgis`
```
Comment (Kelvinrr) [inline README.md:53]: This was using the wrong port compared to what the tests actually use.

**Comment by Kelvinrr** [comment]
Take it back, the function wasn't really used in any tests.

**Pull request merged**